app/models/manageiq/providers/vmware/cloud_manager/orchestration_template.rb

```diff
@@ -7,10 +7,11 @@ def parameter_groups
     )]
 
     # Parse template's OVF file
-    ovf_doc = MiqXml.load(content)
+    ovf_doc = MiqXml.load(content)&.root
```

Comment:
If you want this to be backported recommend staying away from `&.` since it is easy to miss in a cherry-pick and will break on older versions